release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -95,6 +95,18 @@ A new oc command line interface (CLI) flag, `--import-mode`, has been added to t
 == Ingress Node Firewall Operator is generally available
 Ingress Node Firewall Operator was made a technology preview in {product-title} 4.12. With this release, Ingress Node Firewall Operator is generally available. You can now configure firewall rules at the node level. For more information, see xref:../networking/networking-operators-overview.adoc#networking-operators-overview[Ingress Node Firewall Operator].
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`. 
```

Comment:
I know I'm not familiar with the material, but this sentence was hard to follow for me. I can't tell exactly what is new or what is the actual criteria for being able to dynamically used non-reserved CPUs by default. Do they have to be set to static to use this, or are ones set to static considered performance-tuned?

I might not be making much sense here, but maybe see if breaking it up into two sentences would be clearer?